src/PanZoom.tsx

```diff
@@ -79,4 +83,32 @@ const PanZoomWithContext = (props: PanZoomProps, apiRef: PanZoomProps['apiRef'])
   </PanZoomProvider>
 );
 
-export default forwardRef(PanZoomWithContext);
+const PanZoomWithRef = forwardRef(PanZoomWithContext);
+
+const PanZoomWithCover: React.FC<PanZoomProps> = ({
+  cover,
+  zoomMax,
+  width,
+  height,
+  ...props
+}) => {
+  const { ref, size, zoom } = useCover(cover);
+
+  return (
+    <div style={{ width, height }}>
+      <PanZoomWithRef
+        {...props}
+        {...size}
+        ref={ref}
+        boundary={{}}
+        cover={cover}
+        zoomMin={zoom}
```

Comment:
I think more accurate naming to zoom in that case would be ratio / scale

Reply:
yeah you're right